server/src/typeDefs/user.ts

```diff
@@ -41,4 +40,10 @@ export const user = gql`
     password: String!
     name: String
   }
+
+  type UserSetting {
+    id: Int!
+    userId: Int!
```

Comment:
so in the API layer here you shouldnt return userId as part of the UserSettings. that's more of a database way of thinking. generally in graphql you will have nested UserSettings as a child of User when you query it, or even if not, you know from the logged in user who the owning user is (unless you plan on your app to have other users read settings)